rtl/eclrtl/eclregex.cpp

```diff
@@ -1353,6 +1374,9 @@ CCompiledUStrRegExpr* fetchOrCreateCompiledUStrRegExpr(int _regexLength, const U
 
             if (cacheEntry && cacheEntry->hasSamePattern(regexSize, reinterpret_cast<const char *>(_regex), options))
             {
+                // Note a cache hit
+                if (sectionTimer)
+                    sectionTimer->addStatistic(StNumCacheHits, 1);
```

Comment:
This will crash if an old workunit built before this change that takes a timer parameter is run with a new version.
I am not sure of the best solution.  

Probably new regex functions that pass the timer on, and the old regex timer functions pass nullptr